.travis.yml

```diff
@@ -2,8 +2,8 @@
 language: python
```

Comment:
Have you considered using GitHub Actions instead of Travis?

Reply:
I will likely get rid of it, right now we have both..